tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -1422,9 +1463,14 @@ func visitProductPage(timeout time.Duration, wantStatus int, headers ...*header)
 }
 
 // visitWithApp visits the given url by curl in the given container.
-func visitWithApp(url string, pod string, container string, num int) error {
-	cmd := fmt.Sprintf("kubectl exec %s -n %s -c %s -- bash -c 'for ((i=0; i<%d; i++)); do curl -m 0.1 -i -s %s; done'",
+func visitWithApp(url string, pod string, container string, num int, remote bool) error {
+	var cmd string
+	cmd = fmt.Sprintf("kubectl exec %s -n %s -c %s -- bash -c 'for ((i=0; i<%d; i++)); do curl -m 0.1 -i -s %s; done'",
 		pod, tc.Kube.Namespace, container, num, url)
+	if remote {
```

Comment:
to be consistent w/ podID(), consider changing parameter name from 'remote' to 'pod_location' and 'if remote' -> "if pod_location == remoteCluster".
(type passed in probably shouldn't be 'bool' either but no need to change for now ... or ever.)